TLM/TMPE.CitiesGameBridge/Service/NetService.cs

```diff
@@ -102,71 +102,108 @@ public class NetService : INetService {
             IterateNodeSegments(nodeId, ClockDirection.None, handler);
         }
 
-        public void IterateNodeSegments(ushort nodeId,
-                                        ClockDirection dir,
-                                        NetSegmentHandler handler) {
-            NetManager netManager = Singleton<NetManager>.instance;
+        /// <summary>
+        /// Given a <paramref name="nodeId"/>, retrieves the assocaited <see cref="NetNode"/>.
+        ///
+        /// A <c>nodeId</c> of <c>0</c> returns <c>false</c>.
+        /// </summary>
+        /// 
+        /// <param name="nodeId">The id of the segment to retrieve.</param>
+        /// <param name="node">The returned segment.</param>
+        /// 
+        /// <returns>Returns <c>true</c> if successful, otherwise <c>false</c>.</returns>
```

Comment:
Documentation and definition should be on INetservice interface.
I think we should get rid of that interface but before we do ...

More importantly I think now we have duplicate documentation.